docs/user_manual/type_data.md

```diff
@@ -0,0 +1,514 @@
+administrator.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Sample Purposes" %}
+
+Note: This item only applies if your site uses [detailed sample mode](site_configuration.html#detailed_sample_mode).
+
+Sample purpose describes the reason an aliquot was created. Example purposes include, 'Library,' 'Validation,' and
+'Extra.'
+
+The Sample Purposes list page shows all of the available options and offers controls to modify them. To get to this
+page, click "Sample Purposes" in the Institute Defaults list near the bottom of the menu on the left side of the
+screen.
+
+To add new sample purposes, click the "Add" button in the toolbar at the top of the Sample Purposes table. In the
+dialog, enter the quantity of sample purposes that you would like to create, and click the "Create" button. This will
+take you to the bulk Create Sample Purpose page. Enter the names you'd like for the new sample purposes, and click the
+"Save" button at the top right to confirm.
+
+Sample Purposes may only be edited or deleted by MISO administrators. To edit existing purposes, go to the Sample
+Purposes list page, check the checkboxes beside the purposes you would like to edit, and click the "Edit" button in
+the toolbar at the top of the table. This will bring you to the bulk Edit Sample Purpose page, which is identical to
+the Create Sample Purpose page mentioned above. Keep in mind that any changes you make will affect all samples that
+are already using the affected sample purposes.
+
+MISO administrators can also delete sample purposes. To do so, go to the Sample Purposes list page, check the
+checkboxes beside the purposes you would like to delete, and click the "Delete" button in the toolbar at the top of the
+table. You will only be allowed to delete a sample purposes if the option has not been used by any existing samples.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Labs and Institutes" %}
+
+Note: These items only apply if your site uses [detailed sample mode](site_configuration.html#detailed_sample_mode).
+
+If a sample was received from an external lab, this can be recorded at the tissue level. Each lab belongs to an
+institute. Any user may create new labs and institutes in MISO. Administrators may also edit and delete existing labs
+and institutes.
+
+Available institutes can be viewed on the Institutes list page. To get there, click "Institutes" in the Institute
+Defaults list near the bottom of the menu on the left side of the screen. All operations affecting institutes will be
+started from this page.
+
+Available labs can be viewed on the Institutes list page. To get there, click "Labs" in the Institute Defaults list
+near the bottom of the menu on the left side of the screen. All operations affecting institutes will be started from
+this page.
```

Comment:
`Available labs can be viewed on the Labs list page`, and `All operations affecting labs will be started from this page.`